addons/website/static/src/snippets/s_embed_code/options.js

```diff
@@ -36,17 +37,17 @@ class CodeEditorDialog extends Component {
     }
 }
 
-options.registry.EmbedCode = options.Class.extend({
-    init() {
-        this._super(...arguments);
-        this.dialog = this.bindService("dialog");
-    },
+class EmbedCode extends SnippetOption {
+    constructor() {
+        super(...arguments);
+        this.dialog = this.env.services.dialog;
+    }
     //--------------------------------------------------------------------------
     // Options
     //--------------------------------------------------------------------------
 
     async editCode() {
-        const $container = this.$target.find('.s_embed_code_embedded');
```

Comment:
I know last year we talked about also removing jQuery from options, but let's not do that here. Only remove it if you touch the line, as usual. 👍🏻

Reply:
It looked like a quick win here, but ok! :smile: